feature-set/src/lib.rs

```diff
@@ -1179,6 +1183,7 @@ lazy_static! {
         (deplete_cu_meter_on_vm_failure::id(), "Deplete compute meter for vm errors SIMD-0182 #3993"),
         (reserve_minimal_cus_for_builtin_instructions::id(), "Reserve minimal CUs for builtin instructions SIMD-170 #2562"),
         (raise_block_limits_to_50m::id(), "Raise block limit to 50M SIMD-0207"),
+        (raise_block_limits_to_60m::id(), "Raise block limit to 60M SIMD-0256"),
```

Comment:
Usually these go at the bottom, but I guess it's related to the previous feature.